Assets/MRTK/Tools/MSBuild/Scripts/UnityProjectInfo.cs

```diff
@@ -163,7 +163,8 @@ private CSProjectInfo GetProjectInfo(Dictionary<string, CSProjectInfo> projectsM
 
             if (!asmDefInfoMap.TryGetValue(projectKey, out AssemblyDefinitionInfo assemblyDefinitionInfo))
             {
-                throw new InvalidOperationException($"Can't find an asmdef for project: {projectKey}");
+                Debug.LogWarning($"Can't find an asmdef for project: {projectKey}.");
```

Comment:
Do we want to expand the comment re: what may not work?

Reply:
Ended up reverting this change in favor of an existing exclude path with a more verbose comment